pythorhead/mention.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any, Optional
+from pythorhead.requestor import Request, Requestor
+from pythorhead.types import SortType
+
+
+class Mention:
+    def __init__(self, _requestor: Requestor):
+        self._requestor = _requestor
+
+    def list(
+        self,
+        unread_only: bool,
+        sort: Optional[SortType] = None,
+        page: Optional[int] = None,
+        limit: Optional[int] = None,
+    ) -> Optional[dict]:
+        """
+        List all user mentions
+
+        Args: 
+            unread_only (bool).
+            sort? (SortType)
+            page? (int).
+            limit? (int)
+
+        Returns:
+            dict? mentions response
+        """
+        unread_only = 'true' if unread_only else 'false'
```

Comment:
Is this really needed? python automatically parse booleans afaik

Reply:
Yes, when passed into params, the API will complain about it not being a string `true`/`false`. You can try passing it as a bool into the api and see the error it throws.

Reply:
Error:

```
Error encountered while Request.GET: Query deserialize error: provided string was not `true` or `false`
```